### PR TITLE
Normalized the options available in the CLI utilities.

### PR DESCRIPTION
**Pull request by wdschei**
Some of the normalizations were just simple formatting and re-ordering in the source for easier comparison.

### CODE REVIEW

File: cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /***
- *   Copyright 2014 Rackspace US, Inc.
+ *   Copyright 2015 Rackspace US, Inc.
```
Comment (RackerWilliams) [inline cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala]: Don't change the date:

http://stackoverflow.com/questions/2390230/do-copyright-dates-need-to-be-updated

File: cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala [hunk 2]
```diff
@@ -97,6 +97,9 @@ object Wadl2Dot {
   val help = parser.flag[Boolean] (List("h", "help"),
                                    "Display usage.")
 
+  val xpathVersion = parser.option[Int](List("t", "xpath-version"), "n",
```
Comment (RackerWilliams) [inline cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala]: The XPath version should have no effect on the DOT file FYI.  But I guess it's okay to have in there for consistency.

Comment (wdschei) [inline cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala]: It doesn't? I thought it produced the full checker before producing the DOT.

File: cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala (hunk 1 above)
Comment (wdschei) [inline cli/wadl2checker/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Checker.scala]: I change that back.
BTW they want us to change all of them in Repose at the start of the new year.

File: cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala (hunk 2 above)
Comment (wdschei) [inline cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala]: Should I make the help comment state that it isn't used, but is only available for consistency across all the tools?

Comment (wdschei) [inline cli/wadl2dot/src/main/scala/com/rackspace/com/papi/components/checker/cli/Wadl2Dot.scala]: Although the Wadl2Checker doesn't support '-e/--show-errors' or '-n/--nfa-mode' (as it shouldn't), so just removing this one would still be consistent.
I just thought it was an oversight that it wasn't there.

**Comment by wdschei** [comment]
According to the WadlTest.scala it is in the package: `com.rackspace.com.papi.components.checker.cli`
* https://github.com/wdschei/api-checker/blob/NormalizeValidateCliOption/cli/wadltest/src/main/scala/com/rackspace/papi/components/checker/cli/WadlTest.scala#L16

However it is physically on disk at `com/rackspace/papi/components/checker/cli`.
I started to update the directory path, but then I noticed that the package structure used by the entire API-Checker project is `com.rackspace.com.papi`.

Should I move the directory and update the `package` declaration of the other two classes in it?
OR update the package structure of the entire project to `com.rackspace.papi`?

* https://github.com/wdschei/api-checker/blob/NormalizeValidateCliOption/cli/wadltest/src/main/scala/com/rackspace/papi/components/checker/cli/OkayServlet.scala#L16
* https://github.com/wdschei/api-checker/blob/NormalizeValidateCliOption/cli/wadltest/src/main/scala/com/rackspace/papi/components/checker/cli/ValidatorFilter.scala#L16

**Pull request merged**